apps/playground/src/app/quiz/containers/results/results.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { QuizQuestion } from '../../model/QuizQuestion';
+
+
+@Component({
+  selector: 'codelab-quiz-results',
+  templateUrl: './results.component.html',
+  styleUrls: ['./results.component.scss']
+})
+export class ResultsComponent implements OnInit {
+  @Input() answer: string;
+  @Input() question: QuizQuestion;
+  allQuestions: QuizQuestion[];
+  totalQuestions: number;
+  correctAnswersCount: number;
+  percentage: number;
+  completionTime: number;
```

Comment:
I'd group this in an object

Reply:
ok, wrapped them in a quizMetadata object

Reply:
grouped in a quizMetadata object